signwriting/visualizer/visualize.py

```diff
@@ -65,3 +67,35 @@ def signwriting_to_image(fsw: str, antialiasing=True, trust_box=True, embedded_c
                   font=line_font, embedded_color=embedded_color)
 
     return img
+
+
+# pylint: disable=too-many-locals, too-many-arguments
+def signwritings_to_image(fsw_list: List[str], antialiasing: bool = True, trust_box: bool = True,
```

Comment:
"signwritings" is very not elegant imo.
i think this can use the same function as the original, but when encountering a list, call `layout`

```ts
def signwriting_to_image(fsw: Union[str, List[str]], antialiasing=True, trust_box=True, embedded_color=False,
                         line_color: RGBA = (0, 0, 0, 255),
                         fill_color: RGBA = (255, 255, 255, 255), 
                         direction: Literal["horizontal", "vertical"] = "horizontal") -> Image:
    if isinstance(fsw, list):
        images = [
            signwriting_to_image(fsw_string, antialiasing, trust_box, embedded_color, line_color, fill_color)
            for fsw_string in fsw_list
        ]
        return layout_signwriting(images, direction)
   
    .....
```

Reply:
done